roles/confluent.ssl/tasks/provided_keystore_and_truststore.yml

```diff
@@ -46,7 +46,7 @@
       shell: |
         keytool -noprompt -importkeystore \
           -srckeystore {{keystore_path}} \
-          -srcstorepass {{keystore_storepass}} \
+          -srcstorepass {{keystore_storepass}}{% if ssl_keystore_alias|length %} -srcalias {{ssl_keystore_alias}} {% endif %} \
```

Comment:
if ssl_keystore_alias|length returns 0 then the if statement fails? might be more clear to just say ssl_keystore_alias|length == 0

Reply:
exactly - my writing is the abbreviated version - but I like your approach to make it easier to read

Reply:
I will rewrite it with `if ssl_keystore_alias|length > 0`